src/scss/styles.scss

```diff
@@ -832,3 +833,16 @@ img.logo {
 .cursor-move {
     cursor: move !important;
 }
+
+.col-3 {
+    max-width: 240px;
+}
+
+.col-5 {
+    max-width: 420px;
+}
+
+.col-6 {
+    flex: 0 0 calc(100% - 500px);
```

Comment:
Same here, essentially ensuring specificity to the primary content region of the web vault layout instead of any `col-6` class'd element.